build/templates/errors.py.mako

```diff
@@ -1,4 +1,9 @@
+<%
+# Have to put this in a variable and add it that way because mako keeps thinking it is for it, not for the output file
+encoding_tag = '# -*- coding: utf-8 -*-'
+%>\
 #!/usr/bin/python
```

Comment:
We're inconsistent about this, some files have it, some don't. Isn't this useful only for scripts that are meant to be executed from the command line?